front/src/views/Transfers/TransferAgreementOverview/TransferAgreementOverviewView.tsx

```diff
@@ -374,7 +374,13 @@ function TransferAgreementOverviewView() {
     transferAgreementTable = <TableSkeleton />;
   } else {
     transferAgreementTable = (
-      <FilteringSortingTable columns={columns} tableData={graphqlToTableTransformer(data)} />
+      <FilteringSortingTable
+        columns={columns}
+        tableData={graphqlToTableTransformer(data)}
+        initialState={{
+          sortBy: [{ id: "partnerOrg", desc: true }],
```

Comment:
To order alphabetically (A first, Z last), it should be `desc: false` here, right?